lib/src/utils/menu_config.dart

```diff
@@ -68,17 +69,22 @@ class MenuConfig extends InheritedWidget {
     super.key,
     required super.child,
     required this.hasLeading,
+    required this.glideState,
   });
 
   /// Whether the pull-down menu has any [PullDownMenuItem]s with leading
   /// widget such as chevron.
   final bool hasLeading;
 
+  ///
+  // TODO(salvatore): need doc
+  final ValueNotifier<MenuGlideState> glideState;
```

Comment:
Will it be better to make this ValueNotifer global somewhere in glide_state.dart?

Similar to https://github.com/notDmDrl/pull_down_button/blob/main/lib/src/utils/route.dart#L144

And then just use ValueListenableBuilder directly in MenuActionGestureDetector

Reply:
I don't really like global variables, I think we can put glideState and menuAlignmentNotifier in PullDownMenuRoute with no problem

I added a new commit, it's cleaner in my opinion

What do you think? :)

Reply:
Looked at new commit, indeed looks cleaner 👍